File: core/src/remove/tanks/game/application/context/configuration/Configuration.java
package remove.tanks.game.application.context.configuration;

import java.util.Map;

/**
 * @author Mateusz Długosz
 */
public final class Configuration {
    private final Map<String, String> options;

    public Configuration(Map<String, String> options) {
        this.options = options;
    }

    public boolean hasOption(String name) {
        return options.containsKey(name);
    }

    public String getOption(String name) {
        if (!hasOption(name)) {
            throw new OptionNotFoundException(name);
        }
        return options.get(name);
    }
}

File: tools/docs-generator/lib/application/configuration/configuration.py
class Configuration:
    def __init__(self, options):
        self.options = {}
        for key, value in options.items():
            self.options[key] = value

    def get_option(self, name):
        if str(name) in self.options:
            return self.options[str(name)]
        return None
